### DISCUSSION TITLE
How can I convert bs dropdown into bs collapsible ( according )

### DISCUSSION THREAD

**Discussion by Yusufzai** [twbs/bootstrap#37060] [General] [answered]
Is there any solution to destroy the original as it has no effect to the original

As my main goal is the convert the dropdown into collapsible when button is clicked

So, if it is destroying the instance that could be achievable by somehow changing the attribute of the dropdown to collapsible

For more information kindly refer to - https://github.com/twbs/bootstrap/issues/37055

**Comment by mdo** [comment, marked as answer]
Not sure what you're after, but if you need collapsing content, I'd suggest not trying to mix two different behaviors and components and just use the Collapse plugin.
